src/order/zorn.lean

```diff
@@ -202,10 +202,20 @@ have succ_chain (⋃₀ chain_closure) = (⋃₀ chain_closure),
   from (chain_closure_succ_fixpoint_iff chain_closure_closure).mpr rfl,
 h₃ this.symm
 
+end chain
+end zorn
+
+section zorn
+open zorn
+variables {α : Type u} {r : α → α → Prop} {c c₁ c₂ : set α}
+local infix ` ≺ `:50  := r
+local notation `chain` := chain r
+
 /-- Zorn's lemma
 
 If every chain has an upper bound, then there is a maximal element -/
-theorem zorn (h : ∀c, chain c → ∃ub, ∀a∈c, a ≺ ub) (trans : ∀{a b c}, a ≺ b → b ≺ c → a ≺ c) :
+theorem zorn
```

Comment:
Another option is to rename to, e.g., `zorn.exists_maximal_of_chains_bounded`; what do you think about this?

Reply:
I agree with Rob that renaming is best done in another PR. So... shall we add a comment here? Or do we promise to remember?

Reply:
I'll remember. I'll do this tomorrow.

Reply:
In that case, it's simpler to avoid the `end/end/section/...` business and just put `@[nolint]` on it for now.

Reply:
I've followed @rwbarton's suggestion and just put `@[nolint]` and a `-- TODO(lint):` comment.